src/StdChains.sol

```diff
@@ -39,11 +39,19 @@ abstract contract StdChains {
 
     bool private initialized;
 
+    struct ChainData {
```

Comment:
Nit, wdyt of switching the names of the `Chain` and `ChainData` structs? `ChainData` sounds like broader scope and the return data has more info. Def open to keeping this as-is though too. Otherwise this PR LGTM

Reply:
Ah just saw [your comment](https://github.com/foundry-rs/forge-std/pull/268#issuecomment-1375961973), ok that works for me